package.json

```diff
@@ -33,6 +33,8 @@
     "set-yarn-version": "yarn set version stable",
     "enable-corepack": "npm i -g corepack && corepack enable && corepack prepare yarn@4.3.1 --activate",
     "custom-checks": "TS_NODE_PROJECT=./tools/tsconfig.json node --trace-deprecation --experimental-modules --abort-on-uncaught-exception --loader ts-node/esm --experimental-specifier-resolution=node ./tools/custom-checks/run-custom-checks.ts",
+    "tools:install-pre-commit-secret-detection": "pre-commit install && pre-commit autoupdate",
+    "tools:uninstall-pre-commit-secret-detection": "pre-commit uninstall",
```

Comment:
Could you also update our `CONTRIBUTING.md` and / or `BUILD.md` to recommend installing these pre-commits when setting up the environment?